src/dune_config/dune_config.ml

```diff
@@ -413,7 +413,8 @@ let auto_concurrency =
             | pid -> (
               Unix.close fdw;
               let ic = Unix.in_channel_of_descr fdr in
-              let n = input_line ic |> String.trim |> Int.of_string in
+              let n = try input_line ic |> String.trim |> Int.of_string
+                      with _ -> None in
```

Comment:
We want to use the concrete exception here. Moreover, it's best to only surround the function that has the potential to raise here. So:

```
match input_line ic with
| line -> ...
| exception End_of_file -> ..
```

Reply:
Fixed.